### PR TITLE
Improve Ref Docs

### PR DESCRIPTION
**Pull request by sommersoft**
[Per Issue #566](https://github.com/adafruit/circuitpython/issues/566)

No Learn Guides found.

### CODE REVIEW

File: adafruit_max7219/bcddigits.py [hunk 1]
```diff
@@ -66,7 +64,8 @@ def init_display(self):
 
     def set_digit(self, dpos, value):
         """
-        set one digit in the display
+        Set one digit in the display.
```
Comment (kattni) [inline adafruit_max7219/bcddigits.py]: "Display one digit."

File: adafruit_max7219/bcddigits.py [hunk 2]
```diff
@@ -78,7 +77,8 @@ def set_digit(self, dpos, value):
 
     def set_digits(self, start, values):
         """
-        set the display from a list
+        Set the display from a list.
```
Comment (kattni) [inline adafruit_max7219/bcddigits.py]: "Display digits from a list."

File: adafruit_max7219/bcddigits.py [hunk 3]
```diff
@@ -89,7 +89,8 @@ def set_digits(self, start, values):
 
     def show_dot(self, dpos, bit_value=None):
         """
-        set the decimal point for a digit
+        Set the decimal point for a digit.
```
Comment (kattni) [inline adafruit_max7219/bcddigits.py]: "The decimal point for a digit."

File: adafruit_max7219/max7219.py [hunk 4]
```diff
@@ -105,21 +113,23 @@ def brightness(self, value):
 
     def show(self):
         """
-        update the display with recent changes in buffer
+        Updates the display with recent changes in buffer.
```
Comment (kattni) [inline adafruit_max7219/max7219.py]: "Updates the display."

File: adafruit_max7219/max7219.py [hunk 5]
```diff
@@ -105,21 +113,23 @@ def brightness(self, value):
...
         """
         for ypos in range(8):
             self.write_cmd(_DIGIT0 + ypos, self._buffer[ypos])
 
     def fill(self, bit_value):
         """
-        set all buffer bits to a col
+        Sets all buffer bits to a col
```
Comment (kattni) [inline adafruit_max7219/max7219.py]: This should be changed, but I'm not entirely sure what it does, so I'm not entirely sure what to set it to. If you can't think of something reasonable, then leave it.

File: adafruit_max7219/max7219.py [hunk 6]
```diff
@@ -105,21 +113,23 @@ def brightness(self, value):
...
+
         :param bit_value: int value > 0 set the buffer bit, else clears the buffer bit
         """
         self.framebuf.fill(bit_value)
 
     def pixel(self, xpos, ypos, bit_value=None):
         """
-        set one buffer bit
+        Set one buffer bit
```
Comment (kattni) [inline adafruit_max7219/max7219.py:131]: Same as above.

File: adafruit_max7219/max7219.py [hunk 7]
```diff
@@ -128,11 +138,11 @@ def pixel(self, xpos, ypos, bit_value=None):
         self.framebuf.pixel(xpos, ypos, bit_value)
 
     def scroll(self, delta_x, delta_y):
-        """scolls the display using delta_x,delta_y"""
+        """Scolls the display using delta_x,delta_y."""
```
Comment (kattni) [inline adafruit_max7219/max7219.py]: "Scrolls"

**Review by kattni — CHANGES_REQUESTED** [review]
Thanks for doing this! Please resolve the inline reviews. There are two that I don't have suggestions for, and if you can't come up with anything, leave them as they are.

File: adafruit_max7219/max7219.py (hunk 5 above)
Comment (sommersoft) [reply adafruit_max7219/max7219.py]: I think I know what it's doing, but haven't worked with a MAX7219 before so I can't say for sure. I made it a little more understandable, but still vague. I can put in an issue to have mrmcwethy clear up the docstring, and what it does...

File: adafruit_max7219/max7219.py (hunk 6 above)
Comment (sommersoft) [reply adafruit_max7219/max7219.py:131]: I think I know what it's doing, but haven't worked with a MAX7219 before so I can't say for sure. I made it a little more understandable, but still vague. I can put in an issue to have mrmcwethy clear up the docstring, and what it does...

**Review by kattni — APPROVED** [review]

File: adafruit_max7219/max7219.py (hunk 6 above)
Comment (kattni) [reply adafruit_max7219/max7219.py:131]: Sounds good. Thanks!

**Comment by kattni** [comment]
Looks great, thanks!

**Pull request merged**